snippets/get_transaction.py

```diff
@@ -0,0 +1,31 @@
+"""A snippet that walks us through getting a transaction."""
+from xrpl.clients import JsonRpcClient
+from xrpl.models.requests import Ledger, Tx
+
+client = JsonRpcClient("https://s.altnet.rippletest.net:51234/")
```

Comment:
Can you add comments as we go explaining why we're doing things? 

Ex here: # First we connect to the test network